app/controllers/EditionsController.scala

```diff
@@ -85,9 +86,19 @@ class EditionsController(db: EditionsDB,
       .getOrElse(NotFound(s"Issue $id not found"))
   }
 
-  def getAvailableEditions = EditEditionsAuthAction { _ =>
-    Ok(Json.toJson(EditionsTemplates.getAvailableEditions))
-  }
+  def getAvailableEditions = EditEditionsAuthAction { _ => {
+    val allEditions = EditionsTemplates.getAvailableEditions
+    val regionalEditions = allEditions.filter(e => e.editionType == EditionType.Regional)
+    val specialEditions = allEditions.filter(e => e.editionType == EditionType.Special)
+    val trainingEditions = allEditions.filter(e => e.editionType == EditionType.Training)
+    val editions = Map(
```

Comment:
minor: would a `groupBy` here be more future proof?

Reply:
It ought to be, but the strings (eg regionalEditions) need to match the existing values, and it doesn't actually end up any clearer.